### PR TITLE
Move srole to slash commands

### PR DESCRIPTION
**Pull request by thewilloftheshadow**

### CODE REVIEW

File: slashCommands/narratorSlash/sroleSlash.js [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+            {
+                type: "STRING",
+                name: "gamemode",
+                description: "Select the gamemode for this game.",
+                required: true,
+                choices: [
+                    { name: "Quick", value: "quick" },
+                    { name: "Sandbox", value: "sandbox" },
+                    { name: "Custom", value: "custom" },
+                    { name: "Ranked", value: "Ranked" },
```
Comment (Hackante) [inline slashCommands/narratorSlash/sroleSlash.js:20]: `Ranked` (value) should be lower case as the code checks for `ranked`

File: slashCommands/narratorSlash/sroleSlash.js [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+
+        let usedChannels = []
+        db.set(`usedChannels`, usedChannels)
+
+        args = roles ? roles.split(" ") : []
+        args.forEach((arg) => {
+            args[args.indexOf(arg)] = arg.toLowerCase()
+        })
+        if (args.length != alive.members.size && gamemode == "custom") {
+            return interaction.reply("The number of roles do not match the number of players!")
```
Comment (Hackante) [inline slashCommands/narratorSlash/sroleSlash.js:123]: I would suggest to make error messages ephemeral (`interaction.reply({content:"", ephemeral: true})`)

this also applies for lines 265, 269, 273, 277

Comment (Hackante) [reply slashCommands/narratorSlash/sroleSlash.js:123]: If you think this is fine as it is rn. then it would be fine

**Review by Hackante — CHANGES_REQUESTED** [review]
My suggestions are in the file. Please take a look at them and ask me if something is unclear

File: slashCommands/narratorSlash/sroleSlash.js (hunk 2 above)
Comment (thewilloftheshadow) [reply slashCommands/narratorSlash/sroleSlash.js:123]: This was intentional so the supervisors could see the messages as well

Comment (Hackante) [reply slashCommands/narratorSlash/sroleSlash.js:123]: ~~Not me hopeing to hide my faults from soups~~

Comment (inno14) [reply slashCommands/narratorSlash/sroleSlash.js:123]: Lmao

**Review by Hackante — APPROVED** [review]
Ok everything should be fine now

**Pull request merged**